### PR TITLE
Update docs for deprecated prop-types

### PR DESCRIPTION
**Pull request by Slumber86**

### CODE REVIEW

File: docs/rules/no-deprecated.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ React.createClass({ /* Class object */ });
 const propTypes = {
   foo: PropTypes.bar,
 };
+
+import {React PropTypes} from 'react';
```
Comment (ljharb) [inline docs/rules/no-deprecated.md]: This is invalid syntax. Do you mean `import React, { PropTypes } from 'react'`?

Comment (Slumber86) [inline docs/rules/no-deprecated.md]: Missed a comma, 
` import {React, PropTypes} from 'react'; `

Comment (ljharb) [inline docs/rules/no-deprecated.md]: That would import a named import called "React", which doesn't exist.

The syntax I supplied is the correct one.

File: docs/rules/no-deprecated.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ React.createClass({ /* Class object */ });
 const propTypes = {
   foo: PropTypes.bar,
 };
+
+import {React, PropTypes} from 'react';
```
Comment (ljharb) [inline docs/rules/no-deprecated.md]: this is `const { React, PropTypes } = require('react');` which is invalid - `React.React` isn't a thing.

**Pull request merged**